_episodes/advanced-containers.md

```diff
@@ -275,6 +275,12 @@ did in the previous section. You also want to think carefully about size -- if y
 run `docker image ls` you'll see the size of each image all the way on the right of
 the screen. The bigger your image becomes, the harder it will be to easily download.
 
+> ## Security warning
+> Login credentials including passwords, tokens, secure access tokens or other secrets
+> must never be stored in a container. If secrets are stored, they will be found and
+> exploited when made public - either in the near or distant future.
```

Comment:
```suggestion
> must never be stored in a container. If secrets are stored, they are at high risk to be found and
> exploited when made public.
```
Small suggestion for a language edit but I'm okay with either.

Reply:
Done. Thanks for the suggestion.